1-HT-OOP/src/Shape.ts

```diff
@@ -1,3 +1,31 @@
+import { Point } from "./Point";
+
 export abstract class Shape {
+    protected color:string;
+    protected filled:boolean;
+    protected points:Point[];
+    
+    
+    constructor(points?:Point[],color?:string,filled?:boolean);
+    constructor(points:Point[]=[new Point(4,5),new Point(6,8),new Point(6,10)],
+                color:string="green",
+                filled:boolean=true) {
+        //  What is meant by 3 points (2 points is just a line)? 
+        if(points.length<3){
```

Comment:
Shape like rectangular, triangle, polygon, etc has more than 2 points (Assuming that we don't support circle in our code).

2 points is nothing but a straight line (imagine in a graph) 
**(x1,y1)**-----------------------**(x2,y2)**

Resolve this comment if you got the point.